pkgs/development/python-modules/segyio/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, ninja
+, python
+, scikit-build
+, pytest
+, numpy
+}:
+
+stdenv.mkDerivation rec {
+  pname = "segyio";
+  version = "1.9.9";
+
+  postPatch = ''
+    # Removing unecessary build dependency
+    substituteInPlace python/setup.py --replace "'pytest-runner'," ""
+
+    # Fixing bug: This MINSIGSTKSZ is nowhere to be found inside the code base, so I removed this expression
+    substituteInPlace external/catch2/catch/catch.hpp --replace "32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ" 32768
```

Comment:
```suggestion
    substituteInPlace external/catch2/catch/catch.hpp --replace "32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ" "32768"
```

Reply:
It works as is but I agree it makes things clearer.